util/travis/before_install.sh

```diff
@@ -1,49 +1,33 @@
 #!/bin/bash -e
-echo "Preparing for $TRAVIS_COMMIT_RANGE"
 
-if [[ "$LINT" == "1" ]]; then
-	curl http://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-	sudo add-apt-repository "deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-3.9 main"
-	sudo apt-get -yq update
-	sudo apt-get install clang-format-3.9
-	exit 0
-fi
+echo "Preparing for $TRAVIS_COMMIT_RANGE"
 
 . util/travis/common.sh
-needs_compile || exit 0
 
-if [[ $TRAVIS_OS_NAME == "linux" ]]; then
-	sudo apt-get update
-	sudo apt-get install p7zip-full $COMPILER
+if [[ "${LINT}" == "1" ]]; then
+	exit 0
 fi
 
+# needs_compile || exit 0
```

Comment:
oh i didn't removed it because i don't modified sources at the beginning then it doesn't compile anything